### ISSUE TITLE
Rotation resets game state

### ISSUE THREAD

**Issue by Broam** [alaskalinuxuser/app_ship_capt_crew#4]
Switching from portrait to landscape wipes the state of the game. This is a documented Android behavior (wiping state of UI on orientation switch).

The landscape UI experience (at least on a phone) is sub-par. This could be fixed by simply locking to portrait mode.

**Comment by alaskalinuxuser** [comment]
Yes, great point! I should fix that, thanks! This was one of my first apps I made for my android class. I'm still a bit of a beginner, the feedback is helping me learn, thanks.
